src/docs/concepts-development/translations/index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Translations in Flex
+slug: translations
+updated: 2022-04-26
+category: concepts-development
+ingress:
+  This article introduces translations in Flex and how they are modified
+  and edited in Flex Console and the API
+published: true
+---
+
+In the FTW templates, user-facing content is not written directly into
+the source code. Instead, the source code uses the
+[ICU message syntax](https://unicode-org.github.io/icu/userguide/format_parse/messages/)
+that defines variables for each meaningful piece of content, and a
```

Comment:
Hmhm... "variable" feels like a wrong choice of word here. AFAIK, a "message" is another term that is commonly used.

From the ICU doc linked above:

> Messages are user-visible strings, often with variable elements like names, numbers and dates. Message strings are typically translated into the different languages of a UI, and translators move around the variable elements according to the grammar of the target language.

But "variable" is a bit confusing because ICU formatted message can itself contain variables, e.g. `Hi, my name is {name}!` where "name" is the variable.

Reply:
"variable" is indeed ambiguous. a "message" could be one choice, but we talk about translations and translation keys quite often.


> "that defines translation keys for each meaningful message in the UI, and a translator or a content creator can define the specific values for each translation key in their language."

Reply:
Sorry, my PR is a bit confusing as it is based on https://github.com/sharetribe/flex-docs/pull/560 branch. I should have made my PR against that branch for easier reading. But ping @SariSaar as this is really comment on that PR.

Reply:
Thanks, I'll reformulate – the PR was in draft i.e. nowhere near review ready when this branch was made but I'll double check the comments here in case there's still something relevant after the article is actually done

Reply:
Updated in #560